sbin/Release.sh

```diff
@@ -17,30 +17,23 @@ TIMESTAMP="$(date +'%Y%m%d')"
 npm install
 for f in OpenJDK*.tar.gz
 do
-  case $f in
-    *Linux*)
-      OS=Linux && EXT=tar.gz
-      case $f in
-        *x64*)
-        ARCH=x64 ;;
-        *s390x*)
-        ARCH=s390x ;;
-        *ppc64le*)
-        ARCH=ppc64le ;;
-        *aarch64*)
-        ARCH=aarch64 ;;
-    esac ;;
-    *Mac*)
-    OS=Mac && ARCH=x64 && EXT=tar.gz ;;
-    *AIX*)
-    OS=AIX && ARCH=ppc64 && EXT=tar.gz ;;
-  esac
+  re="OpenJDK([a-zA-Z0-9]+)_([a-zA-Z0-9]+)_([a-zA-Z0-9]+)_([a-zA-Z0-9]+)";
+  EXT=tar.gz;
+  if [[ $f =~ $re ]]; 
```

Comment:
suggest using full variable names, e.g. `EXTENSION`, `file` and maybe replacing `re` with the semantics of the pattern you're matching, maybe `fileprefix` or something?

Reply:
Should this regex refactoring also be repeated below?

Reply:
re = regular expression, the entire one I am trying to match, and capture each segment into a variable.  Requires naming convention of builds to be adhered to... (though then again, so did the case statement approach)

Reply:
One other thing to note, this script generates a timestamp upon being run, and it gets used as the one that is published, while the timestamp that is created and added to the build artifact at the time of the build is ignored (is that intentional)?